pkg/process/procutil/process_linux.go

```diff
@@ -793,7 +821,7 @@ func bootTime(hostProc string) (uint64, error) {
 		}
 	}
 
-	return 0, fmt.Errorf("could not parse btime")
+	return 0, fmt.Errorf("btime data does not exist in /proc/stat")
```

Comment:
nit - mind changing to the actual path from [here](https://github.com/DataDog/datadog-agent/pull/8308/files#diff-1a7ef5b19acd0e296b5a6dcc9e7c22493cf50034dd8ffd4df063266308e65cc4R798)?